### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by fabian-beck** [fabian-beck/pure-suggest#214] [Q&A]
What you like and do not like, what features are missing, how you are using the tool, bugs, criticism, ... anything.

**Comment by n00ble108** [comment]
Hi! 
Many thanks for creating the tool. It looks great, somewhat similar to ConnectedPapers but different, somewhat more elaborate, though I have to compare in greater depth with the other visual citations tools to make a more developed comparison. 

A great user experience overall, with useful information about the tool itself. The GUI is a decent compromise between detailed information and synthetic overview, and is both compact and attractive, 

I appreciate that clicking on a citation in the Citation network highlights in the fields Selected / Suggested, and vice versa. 
The tags that describe the suggested citations are also very useful, as are the possibility to filter based on year, search criteria and range of years. 

The keyboard shortcuts / controls can take some learning which is usual for any tool. 

The 'About' pop-up window is highly informative, concise and useful (including the links to similar tools). 

Still, perhaps the GUI could be improved to enable full-window views or enlarged views of the Citation network field, the Selected field and the Suggested field. To be fair, there is a good balance in how the different elements are designed at present. 

Also, the '0' and '1' (cited / cited by) labelling of boxes in the Citation diagram could come as meta-data / properties when one clicks on the boxes-references in the Citation network. The boxes in the Citation network could feature the authors and year instead, or the title of the citation. Alternatively, the boxes in the Suggested citations could be a numerical list (appearing as each of the total number of suggested citations), and appear accordingly in the Citation network.

I understand every design choice has consequences. Compromise is difficult.

In sum, it is a very promising tool that can help beat information overload and keep abreast of emerging knowledge in fast-moving knowledge domains, as well as to explore new domains. It could become even more useful and usable with the ability to further enlarge the different fields (Selected / Suggested / Citation network). 

I look forward to recommending the tool among my research colleagues, networks and in my (semi-) academic blogging about knowledge management and literature reviews. 

Best regards
Ian

**Reply by fabian-beck** [reply]
Thanks for the detailed feedback!

Indeed, users should be able to control the size of the citation network more flexibly; I created a new issue: https://github.com/fabian-beck/pure-suggest/issues/280.

Choosing the labels in the citation network is really a compromise. Author and year are already too long (at least if used for all nodes). I'm not sure if I understand the suggestion about the “numerical list”: numbering the suggestions by position in the list?

What might help regarding the readability of the network is that the next release will feature keyword nodes, which makes interpreting the regions of the network much easier, I believe. Here is a preview:
![grafik](https://user-images.githubusercontent.com/5501795/188786475-012e648d-bda9-4fe1-9369-1caf96ba16e2.png)

**Reply by n00ble108** [reply]
Hi Fabian, 
Yes, the numbers could be numbering the positions in the suggested list. 
Keyword nodes would indeed be interesting!

By extension, and going off topic somewhat, this becomes connected to knowledge graphs as a separate approach to mapping domain-specific knowledge. For the story, I am also trying to see how one could create knowledge graphs to provide a dynamic, holistic state of the art of strands of research and required competencies within related professional roles and activities. I guess it is about trying to make sense of the (over)abundance of information out there, be it from the perspective of academic research and teaching, industry experience and needs, HR/talent acquisition, environmental and social needs, or policy design and evaluation.  Each type of knowledge has its specific focus and sets of activities, but they nonetheless shape each other. Wicked problems like climate change forces people to think outside the box and work together more.
Tools like PURE-Suggest are a fundamental component of such knowledge-driven approach to information management and evidence-based decision-making. My field (spatial planning) is becoming obsessed about 'data-driven decisions', but common sense tells me it should be lived experience and collaboration that drive decisions instead --- not to mention 'wisdom'... 

Ian
